app/assets/stylesheets/layout/_drop_down.sass

```diff
@@ -49,7 +49,6 @@
 .dropdown .dropdown-panel,
 .toolbar .legacy-actions-more
   min-width: 200px
-  max-width: 360px
```

Comment:
I guess we can probably leave it as it is until we notice an issue with too long translations. However, I would argue most people preferred a longer menu instead of clipped off / wrapped items. What do you think?